src/components/Book.jsx

```diff
@@ -15,20 +16,19 @@ const Book = ({ bookObject }) => {
         <p className="book-category">{bookObject.category}</p>
         <strong className="book-title">{bookObject.title}</strong>
         <small className="book-author">{bookObject.author}</small>
-        <button type="submit" className="remove" onClick={handleRemoveBook}>remove</button>
+        <div className="book-btns">
+          <button type="submit" className="remove" onClick={handleRemoveBook}>remove</button>
+          <div className="Line-2"> </div>
+          <button type="submit" className="comment">Comment</button>
+          <div className="Line-2"> </div>
+          <button type="submit" className="edit">Edit</button>
+        </div>
       </div>
-      <div className="book-progress">
-        <img src="" alt="" />
-        <p>
-          70%
-          <br />
-          <span>Completed</span>
-        </p>
-      </div>
-      <div className="porgress-details">
-        <small>CURRENT CHAPTER</small>
-        <p>Chapter 17</p>
-        <button type="submit">UPDATE PROGRESS</button>
+      <BookProgress />
+      <div className="progress-details">
+        <p className="Current-Chapter">CURRENT CHAPTER</p>
+        <p className="Current-Lesson">Chapter 17</p>
+        <div className="Rectangle-2"><span className="Update-progress">Update progress</span></div>
```

Comment:
- [ ]  This button still does not look as on the design, kindly update it again


Yours:
![image](https://github.com/adeola003/bookstore/assets/62705947/8447f076-74c7-417b-92dd-ecb34f8c9e53)
![image](https://github.com/adeola003/bookstore/assets/62705947/db058756-1b55-4f17-802c-87c207a610d9)

Reply:
@GzimAsani  you want me to reduce the width and adjust the padding? Can you be more explicit?

Reply:
- You see the text is not on  center as on the  design, it is much upward

Reply:
Okay